content/en/runtime.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Runtime
+status: Feedback Appreciated
+category: concept
+tags: ["application", "", ""]
+---
+
+## What it is
+
+A runtime, in general, is something that executes a piece of software.
+It is an abstraction of the underlying operating system that knows how to translate the program's commands into respective actions on the operating system level. 
+
+In the context of [cloud native](/cloud-native-apps/), we refer to the container runtime when we talk about the _runtime_. 
+A container runtime specifically implements the [Open Container Initiative](https://opencontainers.org/) specification to ensure consistent handling around the ecosystem. 
+This includes the container image specification, the method by which the runtime can retrieve an image, 
+and how that image is unpacked, layered, mounted, and executed. 
+These runtimes utilize kernel features such as namespaces and cgroups to provide isolation and resource management for the containers.
+
+## Problem it addresses
+
+Container runtimes are a necessary component of container orchestrators such as Kubernetes. 
```

Comment:
This entire paragraph talks about how it helps, not the problem it addresses. It's so easy to fall into that trap. Remember I told you? We immediately think of the solution. 

Should we just move it down?

Reply:
> This entire paragraph talks about how it helps, not the problem it addresses. It's so easy to fall into that trap. Remember I told you? We immediately think of the solution.

You are absolutely right 😅

Moving it down may work out well. The first sentence of the next section is somewhat awkward, and replacing it with this section might be a nice solution.